ddl.dic

```diff
@@ -2865,6 +2868,9 @@ save_
        by explicitly specifying that is adheres to the formal grammar
        provided in SemVer version 2.0.0.
 
+       Added 'unspecified' as a valid units.code for compound data types where
+       the constituent values have differing units.
```

Comment:
This may need to change if the suggestion to generalise the purpose of `unspecified` is accepted.